twilight-http/src/request/scheduled_event/create_guild_scheduled_event/mod.rs

```diff
@@ -245,6 +235,21 @@ impl<'a> AuditLogReason<'a> for CreateGuildScheduledEvent<'a> {
     }
 }
 
```

Comment:
I think the exec method went missing here

Reply:
It's private so it's not breaking, but maybe this struct should just not implement `IntoFuture` at all?

Reply:
Yeah I think this one should only be used through the `stage_instance`/`voice`/`external` so it probably should not be implemented

Reply:
Too bad private trait implementations are not a thing :(